examples/petset/README.md

```diff
@@ -0,0 +1,183 @@
+members as possible. It is recommended to use one or more of the
+[node selection features](http://kubernetes.io/docs/user-guide/node-selection/)
+to schedule PetSet pods into different nodes, and to provide them storage backed
+by independent volumes.
+
+### Scaling Up
+
+MongoDB recommends an odd number of members in a replica set. An admin may
+decide to have, for instance, 5 members in the replica set. Given that there are
+sufficient available persistent volumes, or a dynamic storage provisioner is
+present, scaling up is done with the `oc scale` command:
+
+```bash
+oc scale --replicas=5 petset/mongodb
+```
+
+New pods (containers) are created and they connect to the replica set, updating
+its configuration.
+
+With five members, the scenarios described in the previous section should work
+similarly, though now there is an added resilience to tolerate up to 2 members
+being simultaneously unavailable.
+
+**Note**: scaling up an existing database might require manual intervention. If
+the database size is greater than the `oplogSize` configuration, a manual
+initial sync of the new members will be required. Please consult the MongoDB
+replication manual for more information.
+
+### Scaling Down
+
+An admin may decide to scale down a replica set to save resources or for any
+other reason. For instance, it is possible to go from 5 to 3 members, or from 3
+to 1 member.
+
+While scaling up might be done without manual intervention when the
+preconditions are met (storage availability, size of existing database and
+`oplogSize`), scaling down always require manual intervention.
+
+To scaling down, start with setting the new number of replicas, e.g.:
+
+```bash
+oc scale --replicas=3 petset/mongodb
+```
+
+Note that if the new number of replicas still constitutes a majority of the
+previous number, it is guaranteed that the replica set may elect a new PRIMARY
+in case one of the pods that was deleted had that role. For example, that is the
+case when going from 5 to 3 members.
+
+On the other hand, scaling down to a lower number will temporarily render the
+replica set to have only SECONDARY members and be in read-only mode. That would
+be the case when scaling from 5 down to 1 member.
+
+The next step is to update the replica set configuration to
+[remove members](https://docs.mongodb.com/manual/tutorial/remove-replica-set-member/)
+that no longer exist. This may be improved in the future, a possible
+implementation being setting a PreStop pod hook that inspects the number of
+replicas (exposed via the downward API) and determines that the pod is being
+removed from the PetSet, and not being restarted for some other reason.
+
+Finally, the volumes used by the decommissioned pods may be manually purged.
+Follow the [PetSet documentation](http://kubernetes.io/docs/user-guide/petset/#deleting-a-pet-set)
+for more details on how to clean up after scaling down.
+
+### Known Limitations
+
+* Only MongoDB 3.2 is supported.
```

Comment:
If this is the case, we'll need to update other places where we say you can use the other image versions.

How about the RHEL image for 3.2?